### ISSUE TITLE
Update?

### ISSUE THREAD

**Issue by mfiano** [BorisTheBrave/resynth-tiles#1]
I know this is very old and doesn't appear to be maintained, but could we get support for newer GIMP? It fails to register itself with GIMP as a plugin, unlike the resynthesizer plugin.

**Comment by BorisTheBrave** [comment]
Yes, I haven't looked at this in a long while. I don't have the time to fix it, but I'll accept a PR if you do.

**Comment by bootchk** [comment]
What version of GIMP, what platform, and what is the error message?

I might be able to help, but it would be simpler if you would provide details.

**Comment by bootchk** [comment]
I tested it in GIMP 2.99 on Linux and found no problems.   I did not test on 2.10 on another platform.

I tested because I am developing GimpFu v3 and I use plugins like this one, that I find on the net, to test.
I did make a few minor edits, which is normal when porting from GIMP 2 to 3.
And I uncovered some issues in GimpFu v3, which is not ready for prime time.

Thanks again for sharing.
